src/connectedk8s/azext_connectedk8s/_precheckutils.py

```diff
@@ -136,7 +136,7 @@ def executing_cluster_diagnostic_checks_job(corev1_api_instance, batchv1_api_ins
                     telemetry.set_exception(exception=error_kubectl_delete_helm.decode("ascii"), fault_type=consts.Cluster_Diagnostic_Checks_Release_Cleanup_Failed, summary="Error while executing cluster diagnostic checks Job")
                     return
 
-        chart_path = azext_utils.get_chart_path(consts.Cluster_Diagnostic_Checks_Job_Registry_Path, kube_config, kube_context, helm_client_location, consts.Pre_Onboarding_Helm_Charts_Folder_Name, consts.Pre_Onboarding_Helm_Charts_Release_Name)
+        chart_path = azext_utils.get_chart_path(consts.Cluster_Diagnostic_Checks_Job_Registry_Path, kube_config, kube_context, helm_client_location, consts.Pre_Onboarding_Helm_Charts_Folder_Name, consts.Pre_Onboarding_Helm_Charts_Release_Name, False)
```

Comment:
Why are you explicitly passing new_path as False?

Reply:
The precheck pulls from `consts.Cluster_Diagnostic_Checks_Job_Registry_Path` defined as `mcr.microsoft.com/azurearck8s/helmchart/stable/clusterdiagnosticchecks:0.1.2` which does not have a newly packaged path/version. I actually have no clue in which pipeline/by which Helm version did this chart get packaged. Because the prechecks succeed fine, I can, however, imply that it was done using a new enough Helm version because it can be pulled using Helm 3.12 just like all the newly packaged stuff.

Reply:
So, basically your assumption is that for create, update, upgrade-agents, enable-features and disable-features and disable-cluster-connect, you are checking the min. version and for pre-check you are letting it go without this check.

While this may be fine, I think it is good to confirm why pre-check works.  I believe Abhilash might have some knowledge in this area.  Can you check with him to confirm?

The reason I am asking is because, since we are checking for min version in pre-check, I wonder if may fail in the future and cause CRIs.

Reply:
Ahhh sorry I might need to explain more. The version check there is for the version of the chart being pulled, and for everything but precheck, we want that (dynamically specified) chart version to be greater than or equal to `1.14`. The `clusterdiagnosticchecks` chart path and version are constant as I shared above, namely `mcr.microsoft.com/azurearck8s/helmchart/stable/clusterdiagnosticchecks:0.1.2` i.e. version `0.1.2` and it would be no use to check this constant. Does that make more sense? Will check with Abhilash to see if he knows anything about this particular chart anyways!